setup.py

```diff
@@ -47,6 +47,13 @@
     "docs": ["sphinx>=1.3", "sphinx-bluebrain-theme"],
 }
 
+description = "Search, download, and prepare atlas data."
+long_description = """ Among different sources of data, Allen Brain Institute
+hosts a rich database of gene expression images, Nissl volumes, and annotation atlases.
+The Atlas-Download-Tools library can help you to download single section images
+and entire datasets, as well as the corresponding metadata. It can further
+pre-process the image data to place it in the standard reference space."""
+
```

Comment:
So this will become the text on the PyPI page. I see many repos just read the contents of the `README.md` file here ([appdirs](https://github.com/ActiveState/appdirs/blob/8eacfa312d77aba28d483fbfb6f6fc54099622be/setup.py#L31), [black](https://github.com/psf/black/blob/77d11bb4b4911c91ea97dd43047cf6021d173c65/setup.py#L55-L56))

The only worry I have is that we have a relative path to the logo image that might not work.

Maybe we can leave this for later. I would just add an empty line to separate the paragraphs:
```python
long_description ="""\
Among different sources of data, Allen Brain Institute
hosts a rich database of gene expression images, Nissl volumes, and annotation atlases.

The Atlas-Download-Tools library can help you to download single section images
and entire datasets, as well as the corresponding metadata. It can further
pre-process the image data to place it in the standard reference space.
"""
```

Reply:
Ah, just saw it's just one paragraph in the README. Then we can keep them together.

Reply:
Interesting! I didn't think about doing this! Sounds a good idea for the future!